api/v1/swagger.yaml

```diff
@@ -9596,7 +9596,7 @@ paths:
           name: key
           in: path
           description: >-
-            The id the event to retrieve
+            The id the event to retrieve
```

Comment:
The id of the event to retrieve?

Reply:
im not sure why its registering this as a change